.github/workflows/release.yaml

```diff
@@ -27,17 +27,31 @@ jobs:
           fetch-depth: 0
           ref: ${{ env.TAG_COMMIT }}
       - uses: ./.github/actions/install-deps
-      - uses: ./.github/actions/authenticate-aws
+      - uses: aws-actions/configure-aws-credentials@v1-node16
```

Comment:
Why do we need to create a snapshot on release? This commit should already have a snapshot from the initial merge?

Reply:
Discussed offline, we've separated the release workflow into two separate workflows: release and snapshot.